src/Lean/Server/InfoUtils.lean

```diff
@@ -303,6 +304,9 @@ def Info.docString? (i : Info) : MetaM (Option String) := do
     if let some decl := (← getOptionDecls).find? oi.optionName then
       return decl.descr
     return none
+  | .ofErrorNameInfo eni => do
+    let some errorExplanation := getErrorExplanationRaw? (← getEnv) eni.errorName | return none
+    return errorExplanation.metadata.summary
```

Comment:
Should this contain more of the information that we already store?

Reply:
I think the version numbers might just create noise, but I've added the severity as I did to the completion.